vueapp/src/models/player.ts

```diff
@@ -15,7 +16,9 @@ export class Player {
   }
 
   static generateMockPlayers(n: number, label: string = "Mock Player", start: number = 1): Player[] {
-    return Array.from({ length: n }, (v, i) => i).map(i => new Player(`${label} ${i + start}`))
+    var levelStore = useLevelStore();
+    levelStore.loadLevels();
```

Comment:
This doesn't seem to be the correct way to use the level store. Is it fine to do this every time it's needed or should it be loaded in some central location then accessed through that?

Reply:
Perhaps we can be a little smarter about when to load the levels (an other data)... like if the store detects if the levels have been loaded (perhaps if there are any levels) and load it if necessary.  So `allLevels` become a calculated property and does the checking for us.